setup.py

```diff
@@ -17,6 +17,8 @@
 install_requires = [
     "cytoolz>=0.8.2",
     "ethereum-abi-utils>=0.4.3",
+    "ethereum-keyfile>=0.3.0",
+    "ethereum-keys>=0.1.0-alpha.7",
```

Comment:
This should get moved to a beta....  will try to carve time to do this tomorrow.  We shouldn't release features depending on alpha libraries right (even if they are our own)?